test/integration/imagechange_buildtrigger_test.go

```diff
@@ -42,17 +42,17 @@ func TestSimpleImageChangeBuildTrigger(t *testing.T) {
 		t.Fatalf("Couldn't create ImageRepository: %v", err)
 	}
 
-	created, err := openshift.Client.BuildConfigs(testNamespace).Create(config)
-	if err != nil {
-		t.Fatalf("Couldn't create BuildConfig: %v", err)
-	}
-
-	watch2, err := openshift.Client.BuildConfigs(testNamespace).Watch(labels.Everything(), labels.Everything(), created.ResourceVersion)
+	watch2, err := openshift.Client.BuildConfigs(testNamespace).Watch(labels.Everything(), labels.Everything(), "0")
 	if err != nil {
 		t.Fatalf("Couldn't subscribe to BuildConfigs %v", err)
 	}
 	defer watch2.Stop()
 
+	if _, err := openshift.Client.BuildConfigs(testNamespace).Create(config); err != nil {
```

Comment:
Also, this is less resilient to other problems (because of watch from 0).  I thought ben explained that there really were two changes, so we should test those.